src/controllers/userController.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from 'express';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import Database from '../database';
+import { sendResponse } from '../utils';
+
+dotenv.config();
+
+export const createUser = async (req: Request, res: Response) => {
+  try {
+    const { name, email, phone, address, password } = req.body;
+
+    const userExist = await Database.User.findOne({
+      where: {
+        email,
+      },
+    });
+
+    if (userExist) {
+      return sendResponse<null>(
+        res,
+        400,
+        null,
+        'A user with this email already exists.'
+      );
+    }
+
+    const hashedPassword = await bcrypt.hash(password, 10);
+
+    const user = Database.User.build({
+      name,
+      email,
+      phone,
+      address,
+      password: hashedPassword,
+    });
+
+    await user.save();
+
+    const token = jwt.sign(
+      { id: user.id },
+      process.env.JWT_SECRET || 'secret',
+      { expiresIn: '2h' }
```

Comment:
can you add this secret in config.js because we will use it in reset password to send token

Reply:
the default 'secret'?